src/Orchard.Web/Modules/Orchard.OutputCache/Filters/OutputCacheFilter.cs

```diff
@@ -610,6 +613,14 @@ public class OutputCacheFilter : FilterProvider, IActionFilter, IResultFilter, I
                 }
             }
 
+            //make CacheKey morphable by external modules
+            try {
```

Comment:
This is unnecessary, just let it throw if there is an issue, remove the try/catch.